src/lib/filter-versions.ts

```diff
@@ -0,0 +1,60 @@
+import semverSatisfies from 'semver/functions/satisfies'
```

Comment:
I think this will need the license header otherwise a bot is gonna come in here and yell at us.